tests/tck/features/user/User.feature

```diff
@@ -0,0 +1,265 @@
+Feature: User & privilege Test
+
+  Background: Prepare space
+  Scenario: Create User
+    When executing query:
+      """
+      DROP USER IF EXISTS user1
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      CREATE USER user1 WITH PASSWORD "pwd1"
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      DROP USER IF EXISTS user2
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      CREATE USER user2
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      CREATE USER user1 WITH PASSWORD "pwd1"
+      """
+    Then a ExecutionError should be raised at runtime:
+    When executing query:
+      """
+      CREATE USER IF NOT EXISTS user1 WITH PASSWORD "pwd1"
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      SHOW USERS
+      """
+    Then the result should include:
+      | Account |
+      | "root"  |
+      | "user1" |
+      | "user2" |
+
+  Scenario: Alter user
+    When executing query:
+      """
+      DROP USER IF EXISTS user_tmp
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      ALTER USER user_tmp WITH PASSWORD "pwd1"
+      """
+    Then a ExecutionError should be raised at runtime:
+    When executing query:
+      """
+      CREATE USER IF NOT EXISTS user2 WITH PASSWORD "pwd2"
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      ALTER USER user2 WITH PASSWORD "pwd1"
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      ALTER USER user2 WITH PASSWORD "pwd2"
```

Comment:
It's better to add a validation at here. to check the password was changed successfully. WDYT?
for example 
```
CHANGE PASSWORD user2 FROM "pw2" TO "pwd2"   ---> expected fail
CHANGE PASSWORD user2 FROM "pwd2" TO "pwd3" ---> expected successful
```